backend/preprocessing/hysteresis_loops.py

```diff
@@ -148,8 +146,9 @@ def main():
                         columns=["n_cycles", "hysteresis_area", "stiffness", "creep"]
                     )
 
-                    experiment_metadata.N_fail = 100
+                    experiment_metadata.N_fail = 100  # TODO FIXME!
                     fill_hyst_df(df, experiment_metadata, hyst_df)
+                    hyst_df = hyst_df[:-1]  # Drop last line (it is invalid)
```

Comment:
Thanks for the @sbancal comment! Do you think adding an explanation on why it is invalid would be useful ?